docs/getting-started/kubernetes/kind/index.md

```diff
@@ -0,0 +1,619 @@
+# Getting Started with Kind
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+### Tools
+- `kubectl` (v1.20 or later)
+- `argocd-agentctl` CLI tool ([download from releases](https://github.com/argoproj-labs/argocd-agent/releases)) or you can install locally based on this document
+- `kustomize` (optional, for customization)
+- `kind` you can install locally based on this document
+
+### Agent Planning
+
+- **Unique agent names** following [DNS label standards](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)
+- **Networking plan** for exposing the principal service to agents
+- **Decision on agent mode**: autonomous or managed for each agent
+
+### Install kind
+```bash
+brew install kind
+```
+
+### Clone argocd-agent repository and build CLI
+```bash
+git clone https://github.com/argoproj-labs/argocd-agent.git
+cd argocd-agent
+
+# Build argocd-agentctl CLI
+make build
+
+# Add to PATH (Optional)
+export PATH=$PATH:$(pwd)/dist
+```
+
+## Architecture Overview
```

Comment:
Can you make this diagram more Kind relevant like you gave the Kind clusters some names so maybe they should be marked in the diagram as well?